docs/adopting/use-cases/gitops/gitops-ace.md

```diff
@@ -1,24 +1,29 @@
 # Install App Connect on an existing cluster using GitOps
 
+<!--- cSpell:ignore gitid kubeseal cntk -->
+
 _This is a work in progress, come back for updates._
 
 Steps to install App Connect in an existing cluster using ArgoCD.
 
 ## Pre-requisites
+
 The following is required before proceeding to the next section.
 
 - Provision an OpenShift cluster.
 - IBM Cloud Pak Entitlement Key
 
-
 ## Installation
+
 1. Fork the [multi-tenancy-gitops](https://github.com/cloud-native-toolkit/multi-tenancy-gitops) repository and clone your fork.
-    ```bash
+
+    ```shell
     git clone git@github.com:{gitid}/multi-tenancy-gitops.git
     ```
 
 1. Install the Red Hat OpenShift GitOps operator.
-  ```bash
+
+  ```shell
```

Comment:
I didn't know about "shell"
What is shell? And not use bash?

Reply:
shell and bash pretty much do the same thing as far as content highlighting go.   Shell is a generic for shell script., bash is specific to the bash shell.   For consistency I used shell throughout the rest of the docs.

Reply:
I see, Is “shell" documented in mkdocs or mkdocs material?

I was hoping to find something to deal with windows vs linux on styling

Let's add to he Documentation page where you already have some tips, that we are using shell instead of bash. 

In gatsbyjs page it was “shell script” and I changed them to “bash” 😅

Reply:
I've added a code block section to the docs to explain the content tags and also some advanced features available in code block, such as highlighting and line numbers